package.json

```diff
@@ -136,6 +136,7 @@
     "@types/geopattern": "^1.2.3",
     "@types/graphql": "^14.0.4",
     "@types/jest": "^24.0.11",
+    "@types/jquery": "^3.3.31",
```

Comment:
We don't use jquery.

Reply:
yeah, this was for the tests, but this is all a big WIP, sorry i forgot to add that

Reply:
so it is needed?

Reply:
i dont know, you tell me :)